benchmark/build.gradle

```diff
@@ -1,6 +1,5 @@
 buildscript {
     repositories {
-        jcenter()
```

Comment:
This is an unrelated build fix...for some reason jcenter removed the jmh gradle plugin